src/components/InstructorRevenueCard/index.js

```diff
@@ -0,0 +1,90 @@
+import WrappedRequest from 'components/WrappedRequest'
+import LayoutColumns from 'components/LayoutColumns'
+import currentMonthStartDate from './utils/currentMonthStartDate'
+import totalRevenue from './utils/totalRevenue'
+import removeRevenueMonth from './utils/removeRevenueMonth'
+import RevenuePeriod from './components/RevenuePeriod'
+import LineChart from './components/LineChart'
+
+export default ({revenueUrl}) => revenueUrl
+  ? <WrappedRequest
+      url={revenueUrl}
+      subscribe
+    >
+      {({data}) => {
+        const currentMonthRevenue = find(data, ['month', currentMonthStartDate()])
+        const currentTotalRevenue = totalRevenue(removeRevenueMonth(data, currentMonthStartDate()))
+        const hasCurrentRevenue = size(data) > 0
+          && ((currentMonthRevenue && currentMonthRevenue.revenue > 0) || (currentTotalRevenue && currentTotalRevenue.revenue > 0))
+        const currentMonthDates = map(data, month => month.month)
+        const currentRevenuePoints = map(data, month => month.revenue)
+        const currentMinutesPoints = map(data, month => month.minutes_watched)
+        const revenueColor = '#4786ff' // # blue
+        const minutesColor = '#171e27' // # base-secondary
+
+        if(!hasCurrentRevenue) {
+          return null
+        }
+
+        return (
+          <Card>
+            <div>
+
+              <div className='pa4'>
+                <LayoutColumns 
+                  items={[
+                    <RevenuePeriod
+                      title={<Text 
+                        message='instructorRevenue.previousMonths.title' 
+                        values={[currentTotalRevenue.monthCount]} 
+                      />}
+                      revenue={currentTotalRevenue.revenue}
+                      subscriberMinutes={currentTotalRevenue.minutes_watched}
+                    />,
+                    <RevenuePeriod
+                      title={<Text message='instructorRevenue.currentMonth.title' />}
+                      revenue={currentMonthRevenue.revenue}
+                      subscriberMinutes={currentMonthRevenue.minutes_watched}
+                    />,
+                  ]}
+                />
+              </div>
+
+              <div className='pa4 bg-white-secondary br2 br--bottom'>
+
+                <div>
+                  <LineChart
+                    xAxis={currentMonthDates}
+                    yAxis={[
+                      {
+                        color: revenueColor,
+                        points: currentRevenuePoints,
+                      },
+                    ]}
+                  />
+                </div>
+
+                <div>
+                  <LineChart
+                    xAxis={currentMonthDates}
+                    yAxis={[
+                      {
+                        color: minutesColor,
+                        points: currentMinutesPoints,
+                      },
+                    ]}
+                  />
+                </div>
```

Comment:
This is the public interface for the customized parts of the line charts now, after all the abstracting in `LineChart` of chartjs config